app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,78 @@
+require 'pry'
+
+class TweetsController < ApplicationController
+  get '/tweets' do
+    if !!session[:user_id]
+      @user = User.find(session[:user_id])
+      @tweets = Tweet.all
+      erb :"/tweets/tweets"
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if !!session[:user_id]
+      @user = User.find(session[:user_id])
+      erb :"/tweets/create_tweet"
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do
+    if !params[:content].empty?
+      @user = User.find(session[:user_id])
+      @tweet = Tweet.create(content: params[:content])
+      @user.tweets << @tweet
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```

or alternatively

```
@tweet = current_user.tweets.create(content: params[:content])
```